### ISSUE TITLE
[App Store] Instantly replace "Enable untested app" button by the "Enable" button after an outdated app has been updated

### ISSUE THREAD

**Issue by j-ed** [nextcloud/server#19044]
## Expected behavior
If an app, which is flagged as "untested", is updated with a tested and valid app version, the "Enable untested app" button should instantly be replaced by the "Enable" button.

## Current behavior
If an app, which is flagged as "untested", is updated with a tested and valid app version, the "Enable untested app" button is still displayed. It only vanishes if the list of apps is refreshed.

Example how an updated app is displayed after it has been updated.
![grafik](https://user-images.githubusercontent.com/2572764/72798497-86302900-3c43-11ea-88e9-6f4bf67aec54.png)

After refreshing the app list it is correctly displayed:
![grafik](https://user-images.githubusercontent.com/2572764/72798606-bd9ed580-3c43-11ea-8d66-134840e64e41.png)

## Steps to reproduce
1. Open the Apps menu.
2. Choose an app which is flagged as "untested" and for which an update exists.
3. Update the app and check how the app entry is looking like afterwards.

#### Server Configuration
OS: Linux 4.9.207
Web server: Apache2 2.4.41
Database: MariaDB 10.2.24
PHP version: 7.3.13
Nextcloud version: 18.0.0

#### Client Configuration
Browser: Mozilla Firefox 72.0.1
Operating system: Windows 10

**Comment by DrCarsonBeckett** [comment]
I can confirm that issue. It seems to be like that at least like version 18. Every time since as far as I can recall it was like that.
Just noticed it with version 20 too. Ironically its with the same app ( 😄 ). Good thing is, the app got updated and has now the official version 20 support. But after I updated the app, it still says "enable unsupported/untested version".



Server configuration
Operating system: Debian Buster 10
Web server: Apache/2
Database: MySQL (MariaDB)
PHP version: PHP 7.4
Nextcloud version: 20.0.0.9
Updated from an older Nextcloud/ownCloud or fresh install: Upgraded 19.0.3.1 to 19.0.4.x and then to 20.
Where did you install Nextcloud from: Downloaded the files from the official website and uploaded them manually.
